crates/assists/src/handlers/generate_function.rs

```diff
@@ -36,8 +36,8 @@ use crate::{
 //     bar("", baz());
 // }
 //
-// fn bar(arg: &str, baz: Baz) {
-//     ${0:todo!()}
+// fn bar(arg: &str, baz: Baz) ${0:-> ()} {
```

Comment:
Please just place the cursor here instead of a `-> ()` placeholder.

Reply:
Done :)